pkg/cloud/libvirt/libvirt.go

```diff
@@ -0,0 +1,118 @@
+package libvirt
+
+import (
+	"fmt"
+	"github.com/osbuild/images/pkg/cloud"
+	"io"
+	"os"
+
+	lv "libvirt.org/go/libvirt"
+)
+
+type libvirtUploader struct {
+	connection string
+	pool       string
+	volume     string
+}
+
+func NewUploader(connection string, pool string, volume string) (cloud.Uploader, error) {
+	return &libvirtUploader{
+		connection: connection,
+		pool:       pool,
+		volume:     volume,
+	}, nil
+}
+
+func (lu *libvirtUploader) Check(status io.Writer) error {
+	return nil
+}
+
+func (lu *libvirtUploader) UploadAndRegister(r io.Reader, f *os.File, status io.Writer) (err error) {
+	fmt.Fprintf(status, "Uploading to libvirt...\n")
+
+	// Connect to libvirt
```

Comment:
(nitpick) I'm not a fan of this kind of "what" comments (i.e. comments that explain what is happening in the next few lines). Its pretty clear from the code already and I feel like its not really adding much, so my suggestion would be to drop this (and the ones below that just explain the "what"). Note that I'm a huge fan of "why" comments, i.e. if needed, why something is happening, but here it seems everything is pretty straightforward :)